test/fixtures/lightscript/named-arrow-functions/skinny-white-empty/expected.json

```diff
@@ -0,0 +1,123 @@
+{
+  "type": "File",
+  "start": 0,
+  "end": 9,
+  "loc": {
+    "start": {
+      "line": 1,
+      "column": 0
+    },
+    "end": {
+      "line": 2,
+      "column": 1
+    }
+  },
+  "program": {
+    "type": "Program",
+    "start": 0,
+    "end": 9,
+    "loc": {
+      "start": {
+        "line": 1,
+        "column": 0
+      },
+      "end": {
+        "line": 2,
+        "column": 1
+      }
+    },
+    "sourceType": "script",
+    "body": [
+      {
+        "type": "NamedArrowDeclaration",
```

Comment:
Sorry I didn't think of this before, but we should probably have a test for non-named arrows, eg:

```
=>
```

perhaps plus the slightly more sane
```
f((x) =>)
````